reference/data-types/datatype-type.md

```diff
@@ -11,7 +11,7 @@ Contains information about data types
 | **Type** | **Member** | **Description** |
 | :--- | :--- | :--- |
 | [_string_](datatype-string.md) | **Name** | Type name |
-| [_string_](datatype-string.md) | **TypeMember**[_N_] | Member name based on an internal ID number (based on 1 through _N_, not all values will be used) |
-| [_int_](datatype-int.md) | **TypeMember**[_name_] | Member internal ID number based on _name_ (will be a number from 1 to N) |
+| [_string_](datatype-string.md) | **Member**[_N_] | Member name based on an internal ID number (based on 1 through _N_, not all values will be used) |
```

Comment:
Is this correct?  I didn't look at source.

Reply:
```
No such 'type' member 'TypeMember'
```
https://docs.macroquest.org/reference/top-level-objects/tlo-type/